lib/ui/src/core/context.js

```diff
@@ -50,7 +50,6 @@ export class Provider extends Component {
     // Initialize the state to be the initial (persisted) state of the store.
     // This gives the modules the chance to read the persisted state, apply their defaults
     // and override if necessary
-    this.state = store.getInitialState();
```

Comment:
I'm concerned this will break persistence. The idea (as expressed by the comment above) was that the store's persisted state gets passed into the modules.

Reply:
reverted